### PR TITLE
[tellstick] Add support for suggestion finder

### LINKED ISSUE #15987: [tellstick] Binding doesn´t work any more.  Have to change api url to pa-api.telldus.com
Telldus live have change the api url to pa-api.telldus.com. 

## Expected Behavior
Same as before but api have to change to pa-api.telldus.com.

## Current Behavior
same as before

## Steps to Reproduce (for Bugs)

1.
2.

## Your Environment

* Version used: (e.g., openHAB and add-on versions)
* Environment name and version (e.g. Chrome 76, Java 8, Node.js 12.9, ...):
* Operating System and version (desktop or mobile, Windows 10, Raspbian Buster, ...):

### PR DESCRIPTION
**Pull request by andrewfg**
Split from #16220

Signed-off-by: Andrew Fiddian-Green [software@whitebear.ch](mailto:software@whitebear.ch)

### CODE REVIEW

**Comment by andrewfg** [comment]
@jlaur 4yi

File: bundles/org.openhab.binding.tellstick/src/main/resources/OH-INF/addon/addon.xml [hunk 1]
```diff
@@ -8,4 +8,16 @@
 	<description>This is the binding for Telldus Tellstick Basic, Duo, .Net and Z-Net Lite.</description>
 	<connection>hybrid</connection>
 
+	<discovery-methods>
+		<discovery-method>
+			<service-type>usb</service-type>
+			<match-properties>
+				<match-property>
+					<name>chipId</name>
+					<regex>1781:0C31</regex>
```
Comment (wborn) [inline bundles/org.openhab.binding.tellstick/src/main/resources/OH-INF/addon/addon.xml:17]: Does case matter or not as you also created https://github.com/openhab/org.openhab.binding.zwave/pull/1912?

Comment (andrewfg) [reply bundles/org.openhab.binding.tellstick/src/main/resources/OH-INF/addon/addon.xml:17]: You are correct. Well spotted.

**Comment by andrewfg** [comment]
@wborn I fixed the regex. And I also updated the readme according to #16220. => Probably the latter should be back-ported to v4.1.x ??

**Review by wborn — APPROVED** [review]
Thanks!

**Comment by wborn** [comment]
> Probably the latter should be back-ported to v4.1

IIRC the docs for final releases aren't updated automatically even if PRs are backported.

**Pull request merged**